code/game/objects/items/weapons/mop.dm

```diff
@@ -42,7 +42,7 @@
 
 	var/turf/T = get_turf(A)
 
-	if(istype(A, /obj/item/weapon/reagent_containers/glass/bucket))
+	if(istype(A, /obj/item/weapon/reagent_containers/glass/bucket | /obj/structure/janitorialcart))
```

Comment:
Yeah some testing it still works it would be a cool way to shorted long if istype() chains as incoming said. This can be merged now as it works pretty much 100% but it would be nice if someone with more experience with operator types could tell us their opinion of this.